rfcs/0098-community-team.md

```diff
@@ -0,0 +1,190 @@
+
+The community team will hold its members accountable to these ideals.
+
+## Code of Conduct and Statement of Values
+[values]: #values
+
+Following the adoption of this RFC, the code of conduct and statement of values will be displayed on a dedicated page on the NixOS website. They will read as follows, unless amended by a future RFC.
+
+The Nix community recognizes that its members come from many walks of life, and that it is natural for us to disagree on some things. Nonetheless, we are all committed to certain values, to allow us to work together and build awesome technologies.
+
+Community members are expected to follow these values in their interactions in all official Nix social spaces, including websites, chat rooms, bug threads, and conferences. We have a community team who will set and enforce social norms, to make that easier for everyone. faCommunity members are expected to follow the community team's requests.
+
+If you have opinions on the community team's work, get involved! It's a rotating team, and we want to develop the leadership potential in all community members.
+
+Our values are as follows.
+
+**Collaboration** — We all want to make the best technologies we can. We work together on our common goals, with open discussion, and without petty infighting or focusing on blame. We are all, always, teaching each other to be better.
+
+**Communication** — When we encounter problems, we focus on solving them, not on fighting each other. When we disagree on technical subjects we follow appropriate processes to escalate them to technical leadership. We try to avoid and de-escalate personal conflicts, but when they are unavoidable we seek mediation.
+
+**Empathy** — We try to see perspectives other than our own, to treat everyone with kindness and empathy, to move past mistakes. When we criticise each other we try to do so graciously. We respect each other's different backgrounds and viewpoints, and we respect the effort everyone puts into the community.
+
+**Humility** — We acknowledge that everybody has something to learn, that nobody is infallible.
+
+**Inclusion** — We work to include everybody, especially people from marginalized backgrounds. We strive to be fair and equitable, and to respect the ways people are different. We reject bigotry in all its forms.
+
+**Accommodation** — When community members face barriers to participation, we accommodate their needs to the best of our ability. This includes, but is not limited to, needs related to disability either physical or mental, to medical issues, to religious observations, to parenting, and to sober lifestyles.
+
+**Transparency** — We recognize that the whole community has a stake in decisions. We strive not to be arbitrary, to disclose when important decisions are made, and to make sure there is clear justification.
+
+**Advocacy** — We don't just recite our values, we live by them. We don't simply ignore attacks on our values, we speak up and take action to defend them.
```

Comment:
This feels out of place. I agree with the sentiment, but it seems like a ".... and we mean it!" at the end.

Reply:
On the contrary - I've already cited "Advocacy" elsewhere in this discussion as a justification for why I made another statement. It's the "we speak up" part that's so powerful. If it weren't listed, it would be difficult to make arguments that the values should actually be *used*.

Reply:
advocacy means more than just "... and we mean it!" as you say. without advocacy, we leave room for people to say "well, *i* take these values to heart" while being a bystander to others going against said values.

Reply:
> advocacy means more than just "... and we mean it!" as you say. without advocacy, we leave room for people to say "well, _i_ take these values to heart" while being a bystander to others going against said values.

@IreneKnapp is indicating that these values are for everyone. I think we should allow people to be bystanders if they wish. There may be many reasons someone is unable or unwilling to take action and I tdon't hink they should be faulted for it by default. I would not want someone's general interest in Nix to obligate them to action. The community team, though, has additional obligations to take actions and make decisions at certain times, but we should not fault people in the general community who want to be bystanders.

Reply:
Yeah, I mean, these are things we strive for. I wouldn't go around telling people "do something!", but having the value listed means that if somebody were objecting to an action that somebody *is* taking, they would have a way to push back. Without it there's a bias towards inaction. I've seen this happen in practice in corporate settings.

Reply:
Advocacy is also nice for propagation and ultimately cohesion in combination with communication.

Maybe the twist of the paragraph could better capture the entire spectrum of advocacy that seems to be important here.

Reply:
@blaggacao Would you like to take a shot at writing wording like that? I can't promise to adopt it until I see it, but I would love to see what you're thinking of.

Reply:
I have to admit that I'm not enthusiastic about splitting things into pieces, and I question whether a longer process would actually lead to better results. I'm happy to participate in that process to the extent that you want to drive it, in parallel, but at least right now I am still hopeful that we can move this RFC forward in something approximating its current scope.

I do think that it's a good idea to define how the team this RFC creates relates to the marketing team, even if it's only a brief statement that the marketing team will handle external-facing things and the two teams will coordinate when possible. Thanks for bringing that up.

Reply:
> a longer process would actually lead to better results

A longer process that actually reaches the point where initially opposed sides agree the text is not perfect but the best trade-off that seems to be achievable definitely leads to better results in terms of avoiding damage to the project, and people agreeing to conform to it as a negotiated bargain with alternatives clearly not better.